test/polygons/polygon_corpus.txt

```diff
@@ -1694,3 +1694,272 @@ CondensedMatter64 19 2.30642e-11 1
 -0.06842195833766776 9.6799396338048
 -0.06842195833766818 9.679939633804798
 -0.06842195833766825 9.679939633804798
+DISABLED_Sponge4b 291 5.000000000000000909e-13 27
```

Comment:
The weird triangulation failure. It seems that it requires really high precision to reproduce the error.